@navikt/ds-css/accordion-menu.css

```diff
@@ -1,113 +1,74 @@
-  background: none;
-  cursor: pointer;
-}
-
-.navds-link.navds-accordion-menu__link--active:not(:active):not(:focus) {
-  font-size: 1.125rem;
-  font-weight: 600;
-  color: var(--navds-color-text-primary);
-  text-decoration: none;
+  font-weight: var(--navds-font-weight-bold);
+  color: var(--navds-color-text-link);
+  text-decoration: underline;
 }
 
-.navds-accordion-menu__chevron {
-  margin-right: 0.25rem;
-  font-size: 1rem;
+.navds-accordion-menu-collapsable .navds-accordion-menu__list,
+.navds-accordion-menu-collapsable
+  .navds-accordion-menu-collapsable
+  .navds-accordion-menu-collapsable__button {
+  font-weight: var(--navds-font-weight-regular);
 }
 
-.navds-accordion-menu__button {
+.navds-accordion-menu-item,
+.navds-accordion-menu-collapsable__button {
   display: flex;
-  clear: both;
-  align-items: center;
-  justify-content: space-between;
-  border: 0;
-  width: 100%;
+  padding: 0.625rem 1rem;
+  line-height: 1.5rem; /* TODO: fix typography. */
 }
 
-.navds-accordion-menu__dropdown--content .navds-accordion-menu__link,
-.navds-accordion-menu__dropdown--content .navds-accordion-menu__button {
-  padding-left: calc(var(--navds-layout-padding-small) + 2rem);
+.navds-accordion-menu-collapsable__expand-icon {
+  font-size: 1.5rem;
+  color: var(--navds-color-text-link);
+  flex-shrink: 0;
+  margin-left: 0.5rem;
 }
 
-@media (min-width: 448px) {
-  .navds-accordion-menu__nav {
-    padding: var(--navds-layout-padding-medium-and-larger) 0;
-  }
-
-  .navds-accordion-menu__title {
-    padding: 0 var(--navds-layout-padding-medium-and-larger);
-  }
-
-  .navds-link.navds-accordion-menu__link,
-  .navds-link.navds-accordion-menu__button {
-    padding: 0 var(--navds-layout-padding-medium-and-larger);
-  }
-
-  .navds-accordion-menu__dropdown--content .navds-accordion-menu__link,
-  .navds-accordion-menu__dropdown--content .navds-accordion-menu__button {
-    padding: 0 calc(var(--navds-layout-padding-medium-and-larger) + 2rem);
-  }
+.navds-accordion-menu-collapsable--open
+  > .navds-accordion-menu-collapsable__button
+  > .navds-accordion-menu-collapsable__expand-icon {
+  transform: rotate(180deg);
 }
 
-@media (min-width: 960px) {
-  .navds-accordion-menu__nav {
-    padding: var(--navds-layout-padding-small) 0;
-  }
-
-  .navds-accordion-menu__title {
-    padding: 0 var(--navds-layout-padding-small);
-  }
+.navds-accordion-menu-collapsable .navds-accordion-menu-item,
+.navds-accordion-menu-collapsable
+  .navds-accordion-menu-collapsable
+  .navds-accordion-menu-collapsable__button {
+  padding-left: 2rem;
+}
 
-  .navds-link.navds-accordion-menu__link,
-  .navds-link.navds-accordion-menu__button {
-    padding: 0 calc(var(--navds-layout-padding-small));
-  }
+.navds-accordion-menu-collapsable
+  .navds-accordion-menu-collapsable
+  .navds-accordion-menu-item,
+.navds-accordion-menu-collapsable
+  .navds-accordion-menu-collapsable
+  .navds-accordion-menu-collapsable
+  .navds-accordion-menu-collapsable__button {
```

Comment:
Jeg fant ikke noen enkel løsning på å style innrykk for vilkårlig antall nivåer, så nå har vi denne janky koden her...

Reply:
Enig at det hadde vært fint å funnet en generell løsning på problemet, men jeg har heller ikke funnet noe smart 🤔